controllers/core/configmap_controller.go

```diff
@@ -73,6 +74,17 @@ func (r *ConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		}
 	}
 
+	// Check if branch ENI cooldown period is updated
+	curCoolDownPeriod := cooldown.GetCoolDown().GetCoolDownPeriod()
+	newCoolDownPeriod, err := cooldown.GetVpcCniConfigMapCoolDownPeriodOrDefault(r.K8sAPI, r.Log)
+	if curCoolDownPeriod != newCoolDownPeriod {
+		if err != nil {
```

Comment:
Shouldn't this `err` check come prior to `curCoolDownPeriod != newCoolDownPeriod`

Reply:
But why default? Shouldn't we instead retain the current value available in `curCoolDownPeriod` ?

Reply:
I think you are right. We should stay on what it was if failing on retrieving data from the CM.